src/App.js

```diff
@@ -1,12 +1,13 @@
-import './App.css';
-import MyCalculator from './components/calculator/calculator';
+/* import { Link, Routes, Route } from "react-router-dom"; */
+/* import MyCalculator from './components/calculator/calculator';
 import MyQuotes from './components/quotes/quotes';
+import Home from "./components/home/home"; */
```

Comment:
- [ OPTIONAL ] Please remove these comments. Unused code left as comments make the repo unclean and could cause bugs in the future.